Sources/MySQL/Bind.swift

```diff
@@ -18,7 +22,7 @@ public final class Bind {
     public typealias CBind = MYSQL_BIND
 
     /// MySQL represents Characters using signed integers.
```

Comment:
@collinhundley sounds perfect, I know we generally prefer `[UInt8]` type. @tannernelson can you give final merge word on this one! You have much more MySQL context. 

👍 from me.